test/Plutip.purs

```diff
@@ -136,10 +133,9 @@ suite = do
       withResource (startPlutipServer config)
         (stopChildProcessWithPort config.port) $ const do
         startRes <- startPlutipCluster config unit
-        startRes `shouldSatisfy` case _ of
```

Comment:
Why did you remove the assertion?

Reply:
Because what it was testing is now tested just by the function not throwing an exception